.azuredevops/templates/di-migrate-template.yml

```diff
@@ -0,0 +1,38 @@
+# templates/di-migrate-template.yml
+
+parameters:
+  - name: modelId
+    type: string
+  - name: sourceEndpoint
+    type: string
+  - name: targetEndpoint
+    type: string
+  - name: azureSubscription
+    type: string
+  - name: poolName
+    type: string
+
+jobs:
+  - job: Migrate_Job
+    pool:
+      name: ${{ parameters.poolName }}
+    steps:
+      - task: UseNode@1
+        inputs:
+          version: "16.x"
+
+      - script: |
+          npm install
+        displayName: "Install npm dependencies"
+
+      - task: AzureCLI@2
+        inputs:
+          azureSubscription: ${{ parameters.azureSubscription }}
+          scriptType: "bash"
+          scriptLocation: "inlineScript"
+          inlineScript: |
+            node $(Build.SourcesDirectory)/.azuredevops/scripts/diMigrate.js
```

Comment:
are you going to add a loop for multiple models at once?

Reply:
I decided to do it now